taxonomies/labels.txt

```diff
@@ -17212,17 +17174,47 @@ hu:Lakto-vegetáriánus
 it:Latto-vegetariano
 wikidata:en:Q3496076
 
+# non English needs the English synonyms too so that "European V-Label Végétarien" can be recognized too
+<en:Vegetarian
+en:European Vegetarian Union, EU Vegetarian Union, V-LABEL.EU, European V-LABEL, EU V-LABEL
+ca:Unió Vegetariana Europea
+cs:V-Label Evropské Vegetariánské Unie
+da:Europæiske Vegetarunion, European Vegetarian Union
+de:Europäische Vegetarier-Union
+eo:Eŭropa vegetarana unuiĝo
+es:Unión Vegetariana Europea
+fi:Euroopan vegetaariliitto
+fr:Union Végétarienne Européenne, European Vegetarian Union, EU Vegetarian Union, V-LABEL.EU, European V-Label, EU V-Label
+he:איחוד הצמחונות האירופי
+hu:Európai Vegetáriánus Szövetség, European Vegetarian Union, EU Vegetarian Union, V-LABEL.EU, European V-LABEL, EU V-LABEL
+it:Unione Europea Vegetariani
+nl:Europese vegetarische unie
+sv:Europeiska Vegetariska Unionen
+wikidata:en:Q443112
+
+#https://www.v-label.eu/wp-content/uploads/2017/09/vegetarian_FR_front.png
+<en:European Vegetarian Union
+en:European Vegetarian Union Vegetarian, European Vegetarian Union - Vegetarian, EVU Vegetarian
+da:Europæiske Vegetarunion Vegetarisk, European Vegetarian Union Vegetarisk, Europæiske Vegetarunion - Vegetarisk, European Vegetarian Union - Vegetarisk, EVU Vegetarian
+de:Europäische Vegetarier-Union Vegetarisch, Europäische Vegetarier-Union - Vegetarisch, EVU Vegetarian
+es:Unión Vegetariana Europea Vegetariano, Unión Vegetariana Europea Vegetariana, Unión Vegetariana Europea - Vegetariano, Unión Vegetariana Europea - Vegetariana, EVU Vegetarian
+fr:Union Végétarienne Européenne Végétarien, Union Végétarienne Européenne - Végétarien, EVU Vegetarian
+sv:Europeiska Vegetariska Unionen Vegetarisk, Europeiska Vegetariska Unionen - Vegetarisk, EVU Vegetarian
+
+#https://www.v-label.eu/wp-content/uploads/2016/07/v-label_vegan_rgb.png
 <en:European Vegetarian Union
 <en:Vegan
-en:European Vegetarian Union Vegan
-ca:Unió Vegana Vegetariana Europea
-cs:Veganské označení Evropské Vegetariánské Unie
-de:Europäische Vegetarier-Union Vegan
-es:Unión Vegetariana y vegana Europea
-fi:Euroopan vegetaariliitto vegaaninen, EVU vegaaninen
-fr:Union Végétarienne Européenne - Végétalien
-it:Unione Vegetariana Europea - Vegano
-nl:Europese Vegetarische Unie Vegan, EVU Vegan
+en:European Vegetarian Union Vegan, European Vegetarian Union - Vegan, EVU Vegan
```

Comment:
we now also have the xx: prefix to make something match in all languages, so we can add also:

xx:European Vegetarian Union Vegan, European Vegetarian Union - Vegan, EVU Vegan